include/onnxruntime/core/optimizer/graph_transformer_utils.h

```diff
@@ -25,6 +26,7 @@ std::vector<std::unique_ptr<RewriteRule>> GenerateRewriteRules(TransformerLevel
     and the transformers_and_rules_to_enable. */
 std::vector<std::unique_ptr<GraphTransformer>> GenerateTransformers(TransformerLevel level,
                                                                     gsl::span<const FreeDimensionOverride> free_dimension_overrides,
+                                                                    const IExecutionProvider* cpu_execution_provider,
                                                                     const std::vector<std::string>& rules_and_transformers_to_enable = {});
```

Comment:
As this is required (eventually) by the constant folding it should be a reference to make that clear